src/main/java/org/sefgobal/dataholder/api/CalendarAPI.java

```diff
@@ -33,7 +33,10 @@ public void getUpcomingOnelives(HttpServletRequest request, HttpServletResponse
         formattedDateTime = formattedDateTime.replace(":", "%3A").replace("+", "%2B");
         String url = "https://www.googleapis.com/calendar/v3/calendars/" +
                 oneliveCalendarId +
-                "/events?timeMin=" +
+                "/events" +
+                "?orderBy=startTime" +
+                "&singleEvents=true" +
+                "&timeMin=" +
```

Comment:
Since this is a single string it should be a single line. Could you please fix this? :)

```
oneliveCalendarId +
"/events?orderBy=startTime&singleEvents=true&timeMin=" +
formattedDateTime +
```

Reply:
Done